hindi/joke/joke1.md

```diff
@@ -0,0 +1,26 @@
+एक आदमी स्कूटर पर बैठ कर पिक्चर हाल के सामने
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.